fhir-bulkimportexport-webapp/liberty-config/server.xml

```diff
@@ -0,0 +1,203 @@
+<server description="fhir-server">
+
+    <!-- Enable features -->
+    <featureManager>
+        <feature>localConnector-1.0</feature>
+        <feature>jaxrs-2.1</feature>
+        <!-- <feature>openidConnectClient-1.0</feature> -->
+        <feature>openidConnectServer-1.0</feature>
+        <feature>appSecurity-2.0</feature>
+        <feature>transportSecurity-1.0</feature>
+        <feature>cdi-2.0</feature>
+        <feature>jsonp-1.1</feature>
+        <feature>jsf-2.3</feature>
+        <feature>servlet-4.0</feature>
+        <feature>websocket-1.1</feature>
+        <!-- db2 jdbc driver 4.24.92 doesn't fully implement jdbc-4.2 -->
+        <feature>jdbc-4.1</feature>
+        <feature>mpOpenAPI-1.0</feature>
+        <feature>batchManagement-1.0</feature>
```

Comment:
are there any additional steps required when using this feature?  securing it? adding a truststore? setting a user?

Reply:
yeah, in the sample server.xml, I just reuse the fhir users and the keystores of the fhirserver.
and different roles are configured for javabatch in that sample server.xml.